packages/ui/src/utils/utilsDate.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Converts various date formats to a Date object
+ * @param input - Date input in various formats
+ * @example
+ * convertDate(1677649423) // Date object from Unix timestamp
+ * convertDate('2023-03-01T12:30:23') // Date object from ISO string
+ * convertDate(new Date()) // Returns the same Date object
+ */
+export const convertDate = (input: Date | number | string): Date => {
```

Comment:
Convert doesn't tell me much, e.g. it's used in this file to parse string to date and date to string.
IMO naming them `parse` and `show` would be clearer (i.e. showTime, showTimeAgo, parseDate)